player/command.c

```diff
@@ -2856,6 +2855,10 @@ static int mp_property_sub_text(void *ctx, struct m_property *prop,
     return M_PROPERTY_NOT_IMPLEMENTED;
 }
 
+static int mp_property_sub_text(void *ctx, struct m_property *prop, int action, void *arg) { return get_sub_text(ctx, prop, action, arg, 0); }
+
+static int mp_property_secondary_sub_text(void *ctx, struct m_property *prop, int action, void *arg) { return get_sub_text(ctx, prop, action, arg, 1); }
+
```

Comment:
Same thing I said earlier for these two function. Not one liners, I just wanted the arguments of the function to be spaced the same.